<html>
<head>
	<title></title>
	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
	<link rel="stylesheet" href="./css/normalize.css">
	<link rel="stylesheet" href="./css/milligram.min.css">
	<link rel="stylesheet" href="./css/navbar.css">
	<link rel="stylesheet" href="./css/jquery.qtip.min.css">
	<link rel="stylesheet" href="./css/main.css">
	<link rel="stylesheet" href="./leaflet/leaflet.css" />
	<link rel="stylesheet" href="./leaflet/leaflet.fullscreen.css" />
</head>
<body>
	<main class='wrapper'>
		<nav class="navigation">
			<section class="container">
			   <a class="navigation-title" href="https://pksevolution.github.io/PKS_visualizations">
				  <h1 class="title">Supplemental Website</h1>
			   </a>
			   <ul class="navigation-list float-right">
				  <!--<li class="navigation-item"><a class="navigation-link" href="/list">Sites</a></li>
				  <li class="navigation-item"><a class="navigation-link" href="/login">Login</a></li>-->
			   </ul>
			</section>
		</nav>
		<div class="container">
			<h1>Kingdom-wide analysis of the evolution of the plant type III polyketide synthase superfamily</h1>
			
			<p><strong>Interactive Figure 1.</strong> Network showing synteny between regions containing type III PKS genes. The network contains four 'R-4-C'-enriched syntenic clusters corresponding to CHS function (2, 4, 5, and 14) and four syntenic clusters with LAP5 and LAP6 orthologs (1, 3, 11, and 27). The syntenic clusters showed species-specific distribution (Supplementary Figure 5). Each vertex depicts information on the pPAP-classification of PKS within the genomic region, the number of PKS within the region (number of tandem duplicates) and the cluster membership. Clusters were detected by applying the community detection algorithms, fastgreedy, walktrap, leading eigenvector, and multilevel on the weighted network, followed by affinity propagation clustering. Vertex size reflects degree of the vertex (unweighted number of edges connecting to the vertex).<br />View image <a href="./interactive_figure_01.html">here</a></p>
			
			
			<p><strong>Figure 3.</strong> Phylogenetic tree of type III PKSs with information on syntenic cluster membership, and type of the sequence according to pPAP-classification. The phylogenetic tree, based on amino acid sequences, indicates that the LAP ortholog containing clade and the 'R-4-C'-containing clade evolved by an early duplication event (early GD). The 'R-4-C'-containing syntenic clusters 2/14, 4 and 5 containing clades form highly related but mostly distinct clades in the phylogenetic tree indicating that cluster 2/14, 4 and 5 evolved by duplication events. The LAP5/6 clade contains orthologs of LAP5 and 6 from Arabidopsis thaliana (Supplementary Table S4). Blue arrows indicate 'R-4-A' sequences that evolved independently several times. Blue arrows with star (*) indicate STS sequences of Vitis vinifera, Arachis duranensis, Arachis ipaensis and from Gymnosperms. Magenta arrows indicate duplication events involving (proto) 'R-4-C'-type PKS sequences. Magenta arrows with a star indicate the origin of 'R-4-C' sequences from these events. The gene tree containing 1607 unique sequences was build using RAxML using 1000 bootstrap replications. Supplementary Figure 3 and 4 show information on the taxonomy. The tree shows high transfer bootstrap expectation values (Lemoine <em>et al.</em>, 2018) for all major clades (cf. Supplementary Figure 5). Experimentally validates sequences are indicated by numbers. 1: triketide and tetraketide pyrone synthase, PKS18; 2: phloroglucinol synthase; 3: RppA; 4: quinolone synthase; 5: &beta;-ketoacyl carrier protein synthase III; 6: 2'-oxoalkylresorcylic acid synthase, ORAS; 7: CsyB; 8: 2'-oxoalkylresorcinol synthase, ORS; 9: hydroxyalkyl &alpha;-pyrone synthase, LAP; 10: hydroxyalkyl &alpha;-pyrone synthase, LAP5; 11: hydroxyalkyl &alpha;-pyrone synthase, LAP6; 12: stilbenecarboxylate synthase, SCS; 13: valerophenone synthase, VPS, VPS annotated with 'R-4-C' also show prenylflavonoid synthase function; 14: diketide-CoA synthase, DCS; 15: curcuminoid synthase, CS/CURS; 16: octaketide synthase, OS; 17: chromone synthase; 18: aleosone synthase; 19: pyrrolidine ketide synthase; 20: alkylresorcylic acid synthase, ARS; 21: acridone synthase, ACS; 22: benzalacetone synthase, BAS; 23: olivetol synthase, OLS: 24: 2-pyrone synthase, 2-PS; 25: orcinol synthase; 26: benzophenone synthase, BPS. 
			<br />View image <a href="./figure_03.html">here</a></p>
			
			<p><strong>Figure S01.</strong> Phylogenetic tree for 126 analyzed species. PKS signatures are present in all vascular plants. The species tree was inferred from all genes (STAG) according to Emms and Kelly (2019). Branch length represents the average number of substitution per sites across a gene families. Support value for each bipartition in the consensus STAG tree are the proportion of times that the bipartition is seen in each of the individual species tree estimates. Scale represents substitutions per site. The outgroup was defined by midgroup rooting in FigTree. *: PKS signature present in species as given per OrthoFinder and/or MCL analysis.
			<br />View image <a href="./figure_S_01.html">here</a></p>			
			
			<p><strong>Figure S03.</strong> Phylogenetic tree of type III PKSs with information on syntenic cluster membership, type of the sequence according to pPAP-classification and the taxonomic order. The phylogenetic tree, based on amino acid sequences, indicates that the LAP ortholog containing clade and the 'R-4-C'-containing clade evolved by an early duplication event (early GD). The 'R-4-C'-containing syntenic clusters 2/14, 4 and 5 containing clades form highly related but mostly distinct clades in the phylogenetic tree indicating that cluster 2/14, 4 and 5 evolved by duplication events. The LAP5/6 clade contains orthologs of LAP5 and 6 from Arabidopsis thaliana (Supplementary Table S4). Blue arrows indicate 'R-4-A' sequences that evolved independently several times. Blue arrows with star (*) indicate STS sequences of Vitis vinifera, Arachis duranensis, Arachis ipaensis and from Gymnosperms. Magenta arrows indicate duplication events involving (proto) 'R-4-C'-type PKS sequences. Magenta arrows with a star indicate the origin of 'R-4-C' sequences from these events. The gene tree containing 1607 unique sequences was build using RAxML using 1000 bootstrap replications. Supplementary Figure 4 shows extended information on the taxonomy. Experimentally validated sequences are font-colored in purple for CHSs (indicated by purple star), blue for STSs (indicated by blue star), green for benzalacetone synthase and orange for other PKS sequences. 1: triketide and tetraketide pyrone synthase, PKS18; 2: phloroglucinol synthase; 3: RppA; 4: quinolone synthase; 5: &beta;-ketoacyl carrier protein synthase III; 6: 2'-oxoalkylresorcylic acid synthase, ORAS; 7: CsyB; 8: 2'-oxoalkylresorcinol synthase, ORS; 9: hydroxyalkyl &alpha;-pyrone synthase, LAP; 10: hydroxyalkyl &alpha;-pyrone synthase, LAP5; 11: hydroxyalkyl &alpha;-pyrone synthase, LAP6; 12: stilbenecarboxylate synthase, SCS; 13: valerophenone synthase, VPS, VPS annotated with 'R-4-C' also show prenylflavonoid synthase function; 14: diketide-CoA synthase, DCS; 15: curcuminoid synthase, CS/CURS; 16: octaketide synthase, OS; 17: chromone synthase; 18: aleosone synthase; 19: pyrrolidine ketide synthase; 20: alkylresorcylic acid synthase, ARS; 21: acridone synthase, ACS; 22: benzalacetone synthase, BAS; 23: olivetol synthase, OLS: 24: 2-pyrone synthase, 2-PS; 25: orcinol synthase; 26: benzophenone synthase, BPS. 
			<br />View image <a href="./figure_S_03.html">here</a></p>		
			
			<p><strong>Figure S04.</strong> Phylogenetic tree of type III PKSs with additional information (syntenic cluster membership, type of sequence according to pPAP-classification, number of exons of the gene sequence, and taxonomic information on the family, order, unranked taxonomic information, and class). The gene tree containing 1607 unique sequences was built using RAxML using 1000 bootstrap replications. The tree shows high transfer bootstrap expectation values (Lemoine <em>et al.</em>, 2018) for all major clades.
			<br />View image <a href="./figure_S_04.html">here</a></p>		
			
			<p><strong>Figure S05.</strong> Transfer bootstrap expectation values for phylogenetic gene tree of type III PKSs. A: Phylogenetic tree with overlaid transfer bootstrap expectation values on nodes. The phylogenetic tree shows high bootstrap values on all major clades. B: Distribution of transfer bootstrap expectation values of the complete tree nodes. Transfer bootstrap expectation values were calculated according to Lemoine <em>et al.</em> (2018).
			<br />View image <a href="./figure_S_05.html">here</a></p>	
		</div>
	</main>
	<script src="./leaflet/leaflet.js"></script>
	<script src="./leaflet/leaflet.fullscreen.min.js"></script>
	<script src="./js/jquery-3.5.1.min.js"></script>
	<script src="./js/jquery.qtip.min.js"></script>
	<script src="./js/cytoscape.min.js"></script>
	<script src="./js/cytoscape-qtip.js"></script>
</body>
</html>
